Sources/GutenbergKit/Sources/GBWebView.swift

```diff
@@ -0,0 +1,10 @@
+import WebKit
+
+class GBWebView: WKWebView {
+
+    /// Disables the default bottom bar that competes with the Gutenberg inserter
+    ///
+    override var inputAccessoryView: UIView? {
+        nil
+    }
```

Comment:
For my own edification, is this hiding the toolbar with the previous/next/done buttons above the keyboard while focusing an HTML text input?

Reply:
Yep, but more importantly this dismisses it while using an external keyboard (which messes with the block inserter)